Google.PowerShell/Common/CloudSdkSettings.cs

```diff
@@ -46,7 +49,11 @@ public class CommonProperties
         /// </summary>
         public static string GetCurrentConfigurationName()
         {
+#if !UNIX
```

Comment:
Since we have the same #ifdef in two places, why don't we just create a private helper function, and just ifdef it out once in a more general: `GetCloudSdkFolder`. We can use the result of that to call `Path.Combine` with whatever Cloud SDK files we need.